app/components/EventItem/index.tsx

```diff
@@ -205,12 +201,20 @@ const EventItem = ({
           }}
           className={styles.eventItem}
         >
-          <Flex column gap="var(--spacing-xs)" className="secondaryFontColor">
-            <div>
-              <h3 className={styles.eventItemTitle}>{event.title}</h3>
-              {event.totalCapacity > 0 && <Attendance event={event} />}
-            </div>
-            <TimeStartAndRegistration event={event} />
+          <Flex column gap="var(--spacing-sm)" className="secondaryFontColor">
+            <h3 className={styles.eventItemTitle}>{event.title}</h3>
+            {event.totalCapacity != null && event.totalCapacity > 0 && (
```

Comment:
```suggestion
            {event.totalCapacity !== null && event.totalCapacity > 0 && (
```
I like ===

Reply:
This is actually intentional so to include `undefined` 😄

Reply:
Ah, I thought `!=` would cover more than just `undefined` and `null`, but it seems I was mistaken